src/WP_CLI/SearchReplacer.php

```diff
@@ -114,19 +116,45 @@ private function _run( $data, $serialised, $recursion_level = 0, $visited_data =
 					$search_regex .= $this->regex_delimiter;
 					$search_regex .= $this->regex_flags;
 
-					$result = preg_replace( $search_regex, $this->to, $data, $this->regex_limit );
-					if ( null === $result || PREG_NO_ERROR !== preg_last_error() ) {
+					$check = true; // Set to avoid bogus error on strpos
+
+					if ( $this->callback ) {
+						if ( $check = strpos( $data, $this->from ) !== false ) {
```

Comment:
Given that we are `	if ( $this->regex ) {` , what's the purpose of this `strpos()` check? Shoudn't we just proceed with `call_user_func` ?